api/utils/common.js

```diff
@@ -2283,4 +2283,22 @@ common.checkDatabaseConfigMatch = (apiConfig, frontendConfig) => {
     }
 };
 
-module.exports = common;
\ No newline at end of file
+/**
+ * Sanitizes a filename to prevent directory traversals and such.
+ * @param {string} filename -
+ * @param {string} replacement -
+ * @returns {string} sanitizedFilename -
+ */
+common.sanitizeFilename = (filename, replacement = "") => {
+    if (typeof filename !== "string") {
+        return "";
+    }
+
+    return filename
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected control character(s) in regular expression: \x00, \x1f.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=5176435)